src/file.ts

```diff
@@ -2723,6 +2724,10 @@ class File extends ServiceObject<File> {
    * @param {string} [config.responseDisposition] The
    *     [response-content-disposition parameter](http://goo.gl/yMWxQV) of the
    *     signed url.
+   * @param {*} [config.usableFrom=Date.now()] A timestamp when this link became usable. Any value
```

Comment:
Is the name `usableFrom` readable to you or do you have recommendations for changing it?

Reply:
@frankyn what do you think of `accessibleAt`? (thanks @bcoe for the suggestion!)